RFCs/RFC7.md

```diff
@@ -0,0 +1,24 @@
+# RFC7: Lesson Organizations
+
+## Current vision for Lesson Organizations:
+* Responsibilities: 
+  * Communication with Carpentries: Lesson Organizations will follow all policies as outlined by The Carpentries, 
+  including regular reporting on activities.
+  * Curriculum development: What is a workshop for the Lesson Organization?
+  * Lesson maintenance: What is lesson life cycle (goals of lesson, when does it become supported, when is it retired), 
+  maintaining lesson repositories in GitHub based on the lesson example from The Carpentries GitHub organization, 
+  determining processes associated with lesson maintenance (recommendations for lesson incubation and maintenance?)
+* Governance: Each lesson organization should have a point of contact (preferably a committee of 5-7 people led by a Chair), 
+although the specific methods through which leadership occurs is determined by the lesson organization.
+Onboarding for new lesson organizations: The Carpentries Board will be responsible for assessing alignment of the prospective 
+lesson organization’s mission and their ability to meet the responsibilities described. Given these decisions impact 
+organizational resources, the Board of Directors will solicit community input prior to deciding whether to extend an offer. 
+The Carpentries Board may also recommended compatible administrative structures and policies for prospective Lesson 
+Organizations.
+
```

Comment:
**Responsibilities of the Carpentries to the Lesson Organizations:**
These will continue to be developed, but include the general categories of:
* Workshop coordination, including infrastructure
* Lesson and workshop templates
* Lesson development infrastructure
* Assessment
* Instructor training
* Community support